### ISSUE TITLE
Error when openning empty project

### ISSUE THREAD

**Issue by forsakenlight** [godotengine/godot#8105]
**Operating system or device - Godot version:**
Windows 10 Professional

**Issue description:**

After compile godot from master, I created new project, then when i try to open this project i have got the following error in console:

```
EXEC PATHP??: D:\GameEngines\Godot_Master\godot\bin\godot.windows.tools.64.exe
WARNING: not found: display/window/allow_hidpi
DETECTED MONITORS: 2
Activated GL 3.3 contextgetting extensions
total 361
max ubo light: 409
max ubo reflections: 455  ubo size: 144
EXEC PATHP??: D:\GameEngines\Godot_Master\godot\bin\godot.windows.tools.64.exe
EditorSettings::_get - Warning, not found: filesytem/file_dialog/show_hidden_files
EditorSettings::_get - Warning, not found: import/pvrtc_texture_tool
EditorSettings::_get - Warning, not found: import/pvrtc_texture_tool
GET FORMAT: RGBA8 mipmaps: 1
WARNING: not found: image_loader/filter
WARNING: not found: image_loader/gen_mipmaps
WARNING: not found: image_loader/repeat
EditorSettings::_get - Warning, not found: import/pvrtc_texture_tool
EditorSettings::_get - Warning, not found: import/pvrtc_texture_tool
EditorSettings::_get - Warning, not found: import/pvrtc_texture_tool
EditorSettings::_get - Warning, not found: import/pvrtc_texture_tool
OPENING: D:/GameEnginesProjects/GodotGitMaster/Test3 (D:::GameEnginesProjects::GodotGitMaster::Test3)
EXEC PATHP??: D:\GameEngines\Godot_Master\godot\bin\godot.windows.tools.64.exe
running cmdline: "D:\GameEngines\Godot_Master\godot\bin\godot.windows.tools.64.exe" "-path" "D:/GameEnginesProjects/GodotGitMaster/Test3" "-editor"
EXEC PATHP??: D:\GameEngines\Godot_Master\godot\binE\godot.dwinidotws.toools.r6S4e.tetxien
gs::_get - Warning, not found: import/pvrtc_texture_tool
EditorSettings::_get - Warning, not found: import/pvrtc_texture_tool
DETECTED MONITORS: 2
erasing atlas
erasing atlas
ERROR: There are still MemoryPool allocs in use at exit!
   At: core\dvector.cpp:68
Activated GL 3.3 contextgetting extensions
total 361
max ubo light: 409
max ubo reflections: 455  ubo size: 144
EXEC PATHP??: D:\GameEngines\Godot_Master\godot\bin\godot.windows.tools.64.exe
ERROR: ClassDB::get_parent_class: Condition ' !ti ' is true. returned: StringName()
   At: core\class_db.cpp:272
ERROR: EditorData::get_edited_scene_root: Index current_edited_scene out of size (edited_scene.size()).
   At: editor\editor_data.cpp:599
ERROR: ClassDB::get_parent_class: Condition ' !ti ' is true. returned: StringName()
   At: core\class_db.cpp:272
ERROR: In Object of type 'CheckBox': Attempt to connect nonexistent signal 'button_selected' to method 'CanvasItemEditorViewport._on_select_type'
   At: core\object.cpp:1416
ERROR: In Object of type 'CheckBox': Attempt to connect nonexistent signal 'button_selected' to method 'CanvasItemEditorViewport._on_select_type'
   At: core\object.cpp:1416
ERROR: In Object of type 'CheckBox': Attempt to connect nonexistent signal 'button_selected' to method 'CanvasItemEditorViewport._on_select_type'
   At: core\object.cpp:1416
ERROR: In Object of type 'CheckBox': Attempt to connect nonexistent signal 'button_selected' to method 'CanvasItemEditorViewport._on_select_type'
   At: core\object.cpp:1416
ERROR: In Object of type 'CheckBox': Attempt to connect nonexistent signal 'button_selected' to method 'CanvasItemEditorViewport._on_select_type'
   At: core\object.cpp:1416
ERROR: In Object of type 'CheckBox': Attempt to connect nonexistent signal 'button_selected' to method 'CanvasItemEditorViewport._on_select_type'
   At: core\object.cpp:1416
ERROR: In Object of type 'CheckBox': Attempt to connect nonexistent signal 'button_selected' to method 'CanvasItemEditorViewport._on_select_type'
   At: core\object.cpp:1416
ERROR: RasterizerSceneGLES3::environment_set_tonemap: Condition ' !env ' is true.
   At: drivers\gles3\rasterizer_scene_gles3.cpp:923
ERROR: ClassDB::get_parent_class: Condition ' !ti ' is true. returned: StringName()
   At: core\class_db.cpp:272
ERROR: ClassDB::get_parent_class: Condition ' !ti ' is true. returned: StringName()
   At: core\class_db.cpp:272
ERROR: ClassDB::get_parent_class: Condition ' !ti ' is true. returned: StringName()
   At: core\class_db.cpp:272
ERROR: ClassDB::get_parent_class: Condition ' !ti ' is true. returned: StringName()
   At: core\class_db.cpp:272
ERROR: ClassDB::get_parent_class: Condition ' !ti ' is true. returned: StringName()
   At: core\class_db.cpp:272
ERROR: ClassDB::get_parent_class: Condition ' !ti ' is true. returned: StringName()
   At: core\class_db.cpp:272
ERROR: ClassDB::get_parent_class: Condition ' !ti ' is true. returned: StringName()
   At: core\class_db.cpp:272
ERROR: ClassDB::get_parent_class: Condition ' !ti ' is true. returned: StringName()
   At: core\class_db.cpp:272
ERROR: ClassDB::get_parent_class: Condition ' !ti ' is true. returned: StringName()
   At: core\class_db.cpp:272
ERROR: ClassDB::get_parent_class: Condition ' !ti ' is true. returned: StringName()
   At: core\class_db.cpp:272
ERROR: ClassDB::get_parent_class: Condition ' !ti ' is true. returned: StringName()
   At: core\class_db.cpp:272
ERROR: ClassDB::get_parent_class: Condition ' !ti ' is true. returned: StringName()
   At: core\class_db.cpp:272
ERROR: ClassDB::get_parent_class: Condition ' !ti ' is true. returned: StringName()
   At: core\class_db.cpp:272
ERROR: ClassDB::get_parent_class: Condition ' !ti ' is true. returned: StringName()
   At: core\class_db.cpp:272
ERROR: ClassDB::get_parent_class: Condition ' !ti ' is true. returned: StringName()
   At: core\class_db.cpp:272
ERROR: ClassDB::get_parent_class: Condition ' !ti ' is true. returned: StringName()
   At: core\class_db.cpp:272
ERROR: ClassDB::get_parent_class: Condition ' !ti ' is true. returned: StringName()
   At: core\class_db.cpp:272
ERROR: ClassDB::get_parent_class: Condition ' !ti ' is true. returned: StringName()
   At: core\class_db.cpp:272
ERROR: ClassDB::get_parent_class: Condition ' !ti ' is true. returned: StringName()
   At: core\class_db.cpp:272
ERROR: ClassDB::get_parent_class: Condition ' !ti ' is true. returned: StringName()
   At: core\class_db.cpp:272
ERROR: ClassDB::get_parent_class: Condition ' !ti ' is true. returned: StringName()
   At: core\class_db.cpp:272
ERROR: ClassDB::get_parent_class: Condition ' !ti ' is true. returned: StringName()
   At: core\class_db.cpp:272
ERROR: ClassDB::get_parent_class: Condition ' !ti ' is true. returned: StringName()
   At: core\class_db.cpp:272
ERROR: ClassDB::get_parent_class: Condition ' !ti ' is true. returned: StringName()
   At: core\class_db.cpp:272
ERROR: ClassDB::get_parent_class: Condition ' !ti ' is true. returned: StringName()
   At: core\class_db.cpp:272
ERROR: ClassDB::get_parent_class: Condition ' !ti ' is true. returned: StringName()
   At: core\class_db.cpp:272
ERROR: ClassDB::get_parent_class: Condition ' !ti ' is true. returned: StringName()
   At: core\class_db.cpp:272
WARNING: not found: editor_plugins/enabled
REIMPORT BECAUSE: not previously found
ERROR: ClassDB::get_parent_class: Condition ' !ti ' is true. returned: StringName()
   At: core\class_db.cpp:272
ERROR: ClassDB::get_parent_class: Condition ' !ti ' is true. returned: StringName()
   At: core\class_db.cpp:272
ERROR: ClassDB::get_parent_class: Condition ' !ti ' is true. returned: StringName()
   At: core\class_db.cpp:272
ERROR: ClassDB::get_parent_class: Condition ' !ti ' is true. returned: StringName()
   At: core\class_db.cpp:272
REIMPORTING: res://icon.png
EditorSettings::_get - Warning, not found: import/pvrtc_texture_tool
EditorSettings::_get - Warning, not found: import/pvrtc_texture_tool
1490207812 vs 1490207812
```

**Steps to reproduce:**

**Link to minimal example project:**

**Comment by karroffel** [comment]
Does it crash or anything? Because I get these errors as well *every time*. It's something that needs fixing but is not breaking anything.

**Comment by karroffel** [comment]
can confirm on Linux

**Comment by karroffel** [comment]
This happens on every platform where the editor runs IIRC.

**Comment by forsakenlight** [comment]
karroffel it does not crush (don't exit application), but it freeze with black screen. To exit i have to kill the process or close godot console.

**Comment by Shane-S** [comment]
`ERROR: RasterizerSceneGLES3::environment_set_tonemap: Condition ' !env ' is true.` seems to be because `environment = VS::get_singleton()->environment_create();` happens after `set_tonemapper` in `Environment::Environment` (see [here](https://github.com/godotengine/godot/blob/master/scene/resources/environment.cpp#L1178))

**Comment by vnen** [comment]
What happens in current master? I think some of those messages might've been solved.

**Comment by Shane-S** [comment]
This is the output I get from a debug build of master (on Fedora 25) on opening an empty project:

GLES3: max extensions: 164
hdr supported: 1
GLES3: max ubo light: 409
GLES3: max ubo reflections: 455, ubo size: 144
WARNING: not found: image_loader/filter
WARNING: not found: image_loader/gen_mipmaps
WARNING: not found: image_loader/repeat
OPENING: /data/Documents/Programming/Projects/test (::data::Documents::Programming::Projects::test)
GLES3: max extensions: 164
hdr supported: 1
ERROR: cleanup: There are still MemoryPool allocs in use at exit!
   At: core/dvector.cpp:69.
GLES3: max ubo light: 409
GLES3: max ubo reflections: 455, ubo size: 144
ERROR: environment_set_tonemap: Condition ' !env ' is true.
   At: drivers/gles3/rasterizer_scene_gles3.cpp:923.
ERROR: environment_set_adjustment: Condition ' !env ' is true.
   At: drivers/gles3/rasterizer_scene_gles3.cpp:938.
ERROR: _load: Invalid tag in doc file: InputEvent
   At: editor/doc/doc_data.cpp:880.
ERROR: move_child: Invalid new child position: 2
   At: scene/main/node.cpp:318.
ERROR: get_edited_scene_root: Index current_edited_scene out of size (edited_scene.size()).
   At: editor/editor_data.cpp:600.
ERROR: environment_set_tonemap: Condition ' !env ' is true.
   At: drivers/gles3/rasterizer_scene_gles3.cpp:923.
ERROR: environment_set_adjustment: Condition ' !env ' is true.
   At: drivers/gles3/rasterizer_scene_gles3.cpp:938.
ERROR: move_child: Invalid new child position: 2
   At: scene/main/node.cpp:318.
1024
REIMPORT BECAUSE: not previously found
ERROR: get_child: Index p_index out of size (data.children.size()).
   At: scene/main/node.cpp:1512.
1920
ERROR: get_child: Index p_index out of size (data.children.size()).
   At: scene/main/node.cpp:1512.
REIMPORTING: res://icon.png
ERROR: environment_set_tonemap: Condition ' !env ' is true.
   At: drivers/gles3/rasterizer_scene_gles3.cpp:923.
ERROR: environment_set_adjustment: Condition ' !env ' is true.
   At: drivers/gles3/rasterizer_scene_gles3.cpp:938.
ERROR: environment_set_tonemap: Condition ' !env ' is true.
   At: drivers/gles3/rasterizer_scene_gles3.cpp:923.
ERROR: environment_set_adjustment: Condition ' !env ' is true.
   At: drivers/gles3/rasterizer_scene_gles3.cpp:938.
ERROR: get_child: Index p_index out of size (data.children.size()).
   At: scene/main/node.cpp:1512.
ERROR: get_child: Index p_index out of size (data.children.size()).
   At: scene/main/node.cpp:1512.

**Comment by TheShed412** [comment]
I'm new here but I noticed the same thing. 
Running Windows 7 Ultimate 64 bit

I started an existing project so some errors are related to that I think. 

>DETECTED MONITORS: 2
Activated GL 3.3 contextGLES3: max extensions: 284
hdr supported: 1
GLES3: max ubo light: 4096
GLES3: max ubo reflections: 1024, ubo size: 144
EditorSettings::_get - Warning, not found: import/pvrtc_texture_tool
EditorSettings::_get - Warning, not found: import/pvrtc_texture_tool
OPENING: C:/godot/bin/test (C:::godot::bin::test)
running cmdline: "C:\godot\bin\godot.windows.tools.32.exe" "-path" "C:/godot/bin
/test" "-editor"
EditorSettDiEnTgEsC:T:E_Dg eMtO N-I TWOaRrSn:i n2g
 not found: import/pvrtc_texture_tool
EditorSettings::_get - Warning, not found: import/pvrtc_texture_tool
ERROR: There are still MemoryPool allocs in use at exit!
   At: core\dvector.cpp:69
Activated GL 3.3 contextGLES3: max extensions: 284
hdr supported: 1
GLES3: max ubo light: 4096
GLES3: max ubo reflections: 1024, ubo size: 144
ERROR: RasterizerSceneGLES3::environment_set_tonemap: Condition ' !env ' is true
.
   At: drivers\gles3\rasterizer_scene_gles3.cpp:923
ERROR: RasterizerSceneGLES3::environment_set_adjustment: Condition ' !env ' is t
rue.
   At: drivers\gles3\rasterizer_scene_gles3.cpp:938
ERROR: Invalid tag in doc file: InputEvent
   At: editor\doc\doc_data.cpp:880
ERROR: Invalid new child position: 2
   At: scene\main\node.cpp:318
ERROR: EditorData::get_edited_scene_root: Index current_edited_scene out of size
 (edited_scene.size()).
   At: editor\editor_data.cpp:600
unicode error: invalid skip
ERROR: RasterizerSceneGLES3::environment_set_tonemap: Condition ' !env ' is true
.
   At: drivers\gles3\rasterizer_scene_gles3.cpp:923
ERROR: RasterizerSceneGLES3::environment_set_adjustment: Condition ' !env ' is t
rue.
   At: drivers\gles3\rasterizer_scene_gles3.cpp:938
ERROR: Invalid new child position: 2
   At: scene\main\node.cpp:318
1920
ERROR: Node::get_child: Index p_index out of size (data.children.size()).
   At: scene\main\node.cpp:1512
ERROR: RasterizerSceneGLES3::environment_set_tonemap: Condition ' !env ' is true
.
   At: drivers\gles3\rasterizer_scene_gles3.cpp:923
ERROR: RasterizerSceneGLES3::environment_set_adjustment: Condition ' !env ' is t
rue.
   At: drivers\gles3\rasterizer_scene_gles3.cpp:938
EditorSettings::_get - Warning, not found: import/pvrtc_texture_tool
EditorSettings::_get - Warning, not found: import/pvrtc_texture_tool
ERROR: Node::get_child: Index p_index out of size (data.children.size()).
   At: scene\main\node.cpp:1512

So I end up with that
>ERROR: RasterizerSceneGLES3::environment_set_tonemap: Condition ' !env ' is true.

that @Shane-S ended up getting
